pkg/doc/verifiable/example_presentation_test.go

```diff
@@ -18,7 +18,7 @@ import (
 //nolint:gochecknoglobals
 var privHolderKey = ed25519.PrivateKey{137, 207, 34, 115, 168, 80, 44, 145, 236, 206, 165, 152, 211, 195, 240, 205, 232, 37, 216, 101, 58, 133, 198, 107, 232, 119, 30, 80, 176, 137, 10, 251, 109, 144, 158, 93, 189, 195, 0, 24, 15, 29, 166, 185, 169, 69, 246, 25, 182, 179, 115, 54, 107, 4, 123, 30, 5, 88, 175, 94, 109, 15, 34, 113} //nolint:lll
 
-func ExamplePresentation() {
+func ExampleNewPresentation() {
```

Comment:
Since this isn't new code, this is optional - but, I noticed that this file ends in "_test" but doesn't seem to actually have tests in it? I wonder if it's possible to either turn these examples in unit tests... or perhaps just remove the word "test" from the file name.

Reply:
@DRK3 Example prefixes in Go are also counted as tests.

They create testable documentation.

Reply:
Gotcha - good to know!